src/Faker/Provider/en_US/Company.php

```diff
@@ -90,4 +99,18 @@ public function bs()
 
         return join($result, ' ');
     }
+
+    /**
+     * Employer Identification Number (EIN)
+     *
+     * @link https://en.wikipedia.org/wiki/Employer_Identification_Number
+     * @example '12-3456789'
+     */
+    public static function ein()
+    {
+        $prefix = static::randomElement(static::$einPrefixes);
+        $suffix = static::numberBetween(0, 9999999);
```

Comment:
This is probably not what you want, since it can be `1` or `4567`, and not only `3456789`. To have exactly 7 digits, use `static::numerify('#######')`.

Reply:
Since I'm composing the string with `sprintf()`, the suffix gets left-padded `$suffix` with zeroes (`%07d`), but I'm happy to make the change if you'd prefer me to use `numerify()` instead?

Reply:
oh, you're right. No need to change then.

Reply:
No worries, thanks for the hip check!